pyproject.toml

```diff
@@ -139,6 +139,7 @@ filterwarnings = [
     'ignore:.*numpy.ufunc size changed.*:RuntimeWarning',
     'ignore:.*Given trait value dtype "float64":UserWarning',
     'ignore:.*The NumPy module was reloaded*:UserWarning',
+    'error::pyvista.PyVistaDeprecationWarning'
```

Comment:
I added this warning conversion to an error in oder to detect which docstrings use deprecated methods (when using `pytest --doctest-module`), but also that warnings are assessed in tests.

Let me know if it is fine for you.

Reply:
Ahh this is great, thanks!